server/src/main/java/com/google/coffeehouse/StorageHandler.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.coffeehouse;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+
+public class StorageHandler {
+
+  static void getPersonQuery(DatabaseClient dbClient, String userId) {
```

Comment:
(Drive-by review comment) Just so this file doesn't get gigantic it might be worth putting this in the Person class instead, and then having StorageHandler just have a method like `.executeSql()`. Just a thought, feel free to leave it how it is. I have a personal thing about super long files but that's more of a me problem than a code problem lol